tests/unittests/test_window.py

```diff
@@ -0,0 +1,77 @@
+import subprocess
+import time
+from unittest import TestCase
+
+from ahk import AHK
+
+
+class TestWindow(TestCase):
+
+    def setUp(self):
+        self.ahk = AHK()
+        self.p = subprocess.Popen('notepad')
+        time.sleep(1)
+        self.win = self.ahk.win_get(title='Untitled - Notepad')
+        self.assertIsNotNone(self.win)
+
+    def test_transparent(self):
+        self.assertEqual(self.win.transparent, 255)
+
+        self.win.transparent = 220
+        self.assertEqual(self.win.transparent, 220)
+
+        self.win.transparent = 255
+        self.assertEqual(self.win.transparent, 255)
+
+    def test_pinned(self):
+        self.assertFalse(self.win.always_on_top)
+
+        self.win.always_on_top = True
+        self.assertTrue(self.win.always_on_top)
+
+        self.win.always_on_top = False
+        self.assertFalse(self.win.always_on_top)
+
+    def test_close(self):
+        self.win.close()
+        self.assertFalse(self.win.exist)
+
+    def test_show_hide(self):
+        self.win.hide()
+        self.assertFalse(self.win.exist)
+
+        self.win.show()
+        self.assertTrue(self.win.exist)
+
+    def test_kill(self):
+        self.win.kill()
+        self.assertFalse(self.win.exist)
+
+    def test_max_min(self):
+        self.assertTrue(self.win.non_max_non_min)
+
+        self.win.maximize()
+        self.assertTrue(self.win.maximized)
+
+        self.win.minimize()
+        self.assertTrue(self.win.minimized)
```

Comment:
😕 AppVeyor throws this exception but in my computer all test are succeed

```py
self = <test_window.TestWindow testMethod=test_max_min>

    def test_max_min(self):
        self.assertTrue(self.win.non_max_non_min)
    
        self.win.maximize()
        self.assertTrue(self.win.maximized)
    
        self.win.minimize()
>       self.assertTrue(self.win.minimized)
E       AssertionError: False is not true

tests\unittests\test_window.py:57: AssertionError
````

Reply:
Yeah, this works locally for me, too. I'll run the tests interactively in AppVeyor and see what I can find out.

Reply:
I retried the build twice, it passed the second time. 

Perhaps there's a race condition that the build server is more prone to than our desktops. Maybe adding a small sleep just after `self.win.minimize()` will help.

Reply:
Looks like it's not always that part that fails, either.

```py
self = <test_window.TestWindow testMethod=test_show_hide>
    def test_show_hide(self):
        self.win.hide()
        self.assertFalse(self.win.exist)
    
        self.win.show()
>       self.assertTrue(self.win.exist)
E       AssertionError: False is not true
tests\unittests\test_window.py:44: AssertionError

```

Reply:
Repeating the tests locally over and over also eventually reveals a failure. To reproduce eventually:

```py
import subprocess
while True:
    subprocess.run(['python', '-m', 'pytest', 'tests\\unittests\\test_window.py'], check=True)
```